app/Listeners/QueueMessage.php

```diff
@@ -33,20 +33,18 @@ public function handle(QueueMessageRequest $event)
         $resources = $event->resources;
 
         // Build the email.
-        $email = new Email($resources['message'], $resources['contest'], $resources['competition'], $resources['users']);
-
-        foreach ($email->allMessages as $message) {
-            $content = $message['message'];
+        $email = new Email($resources);
```

Comment:
Yeah, so I think this is what's causing you trouble! You can only do dependency injection "magically" in the constructor if Laravel is creating the class for you (through injection in another class or the `app()` helper or something like that). So when you add `Manager $manager` to the constructor on line 50 in `Email.php` it expects that parameter to be passed here.